packages/core/src/hooks/useEthers.ts

```diff
@@ -45,12 +45,16 @@ export function useEthers(): Web3Ethers {
     account: accounts[0],
     active: !!provider,
     activate: async (providerOrConnector: SupportedProviders) => {
-      if ('getProvider' in providerOrConnector) {
-        console.warn('Using web3-react connectors is deprecated and may lead to unexpected behavior.')
-        await providerOrConnector.activate()
-        return activate(await providerOrConnector.getProvider())
+      try {
+        if ('getProvider' in providerOrConnector) {
+          console.warn('Using web3-react connectors is deprecated and may lead to unexpected behavior.')
+          await providerOrConnector.activate()
+          return activate(await providerOrConnector.getProvider())
+        }
+        return activate(providerOrConnector)
+      } catch (error) {
+        console.error(error)        
```

Comment:
I don't think we should catch the errors in activate but rather allow the user code to handle them

Reply:
True, removed error catching